playbook.yml

```diff
@@ -7,17 +7,14 @@
     galaxy_config_style: yaml
     galaxy_layout: legacy-improved
     galaxy_server_dir: /galaxy/server
-    galaxy_commit_id: release_20.09
-    galaxy_repo: https://github.com/galaxyproject/galaxy.git
-    galaxy_clone_depth: 1
     galaxy_virtualenv_command: /usr/bin/python3 -m virtualenv
     galaxy_virtualenv_python: python3
     pip_extra_args: "--no-cache-dir --compile"
     galaxy_manage_database: false
+    galaxy_manage_clone: false
+    galaxy_manage_existing: true
     galaxy_client_make_target: client-production
     galaxy_config_files:
-      - src: files/job_conf.xml
-        dest: "{{ galaxy_server_dir }}/config/job_conf.xml"
```

Comment:
Without this file (and it defining K8s as a runner), `ansible-galaxy` will not install the K8s dependencies into Galaxy's virtualenv on the image.

Reply:
Thanks for catching that. Fixed.